lib/msal-core/src/telemetry/TelemetryUtils.ts

```diff
@@ -0,0 +1,35 @@
+import { B2cAuthority } from "../B2cAuthority";
+import { TrustedHostList } from "../Constants";
+import { Utils } from "../Utils";
+
+// This is used to replace the real tenant in telemetry info
```

Comment:
Can be moved to TelemetryConstant.ts may be?

Reply:
updating

Reply:
updated